pkg/gotohelm/testdata/src/example/main.go

```diff
@@ -39,6 +39,10 @@ func main() {
 			out = map[string]any{}
 		}
 
+		if err != nil {
+			err = fmt.Sprintf("%+v", err)
```

Comment:
Why `fmt.Sprintf`?

Reply:
The JSON serialization of error sometimes is pretty worthless. Printf turns it into a human readable string that we can then show in the tests. I had a out of bounds error happening for a bit but the JSON version of it didn't contain any usable information it was just {}. The sprintf version had the full stacktrace